java/Mekhmet-Efe_Adzhar/src/com/zoolatech/lecture1/tasks/_1/Number1.java

```diff
@@ -0,0 +1,147 @@
+        resultPanel = new JPanel();
+        textField = new JTextField(10);
+        frame = new JFrame();
+        float result = 0F;
+
+        //SETTING UP APP INTERFACE WITH SWING FRAMEWORK:
+        //MARK: BUTTONS
+        buttonForKilometer.setBounds(0, 0, 50, 50);
+        buttonForKilometer.setText("km");
+        buttonForMiles.setBounds(0, 0, 50, 50);
+        buttonForMiles.setText("mile");
+        resultButton.setBounds(170, 195, 80, 30);
+        resultButton.setText("Result");
+
+        //MARK: LABELS
+        labelForNumberPanel = new JLabel
+                ("<html><body>Write down a distance that you<br>wanna convert to something</body></html>", JLabel.CENTER);
+        labelForChoiсePanel = new JLabel
+                ("<html><body>Choose unit of distance you<br>wanna to convert to something</body></html>", JLabel.CENTER);
+        labelForResultPanel = new JLabel
+                ("Results are: " + result, JLabel.CENTER);
+
+        // MARK: FIRST TEXT FIELD TO WRITE DOWN A NUMBER TO CONVERT
+        textField.setSize(200, 50);
+        textField.setBorder(BorderFactory.createBevelBorder(30, Color.black, Color.BLUE));
+
+        //MARK: NUMBER PANEL. USER INPUTS A NUMBER WHICH HE/SHE/IT WANNA TO CONVERT.
+        numberPanel.setBackground(Color.CYAN);
+        numberPanel.setBounds(80, 20, 250, 80);
+        numberPanel.add(labelForNumberPanel, BorderLayout.NORTH);
+        numberPanel.add(textField, BorderLayout.SOUTH);
+
+        //MARK: CHOICE PANEL. USER CHOOSE A UNIT OF DISTANCE TO CONVERT HIS/HER/ITS NUMBER.
+        choicePanel.setBackground(Color.ORANGE);
+        choicePanel.setBounds(80, 110, 250, 80);
+        choicePanel.add(labelForChoiсePanel, BorderLayout.NORTH);
+        choicePanel.add(buttonForKilometer, BorderLayout.SOUTH);
+        choicePanel.add(buttonForMiles, BorderLayout.SOUTH);
+
+        //MARK: RESULT PANEL. USER CAN SEE THE RESULTS OF THE CALCULATION :)
+        resultPanel.setBackground(Color.yellow);
+        resultPanel.setBounds(80, 230, 250, 30);
+        resultPanel.add(labelForResultPanel, BorderLayout.NORTH);
+
+        //MARK: FRAME. SOMETHING LIKE A VIEW :)
+        frame.setResizable(false);
+        frame.setSize(400, 300);
+        frame.add(numberPanel);
+        frame.add(choicePanel);
+        frame.add(resultPanel);
+        frame.add(resultButton);
+        frame.setDefaultCloseOperation(WindowConstants.EXIT_ON_CLOSE);
+        frame.setTitle("Convert Everything");
+        frame.setLayout(null);
+        frame.setVisible(true);
+        //MARK: SETTING UP ENDED
+
+        resultButton.addActionListener(e -> buttonAcction());
+    }
+
+    //MARK: LOGICS
+    public void buttonAcction() {
+
+        //MARK: SOME MAGIC HERE!
+        String text = textField.getText();
+        result = Float.parseFloat(text);
+
+        if (buttonForKilometer.getModel().isSelected()) {
+            labelForResultPanel.setText("Results are: " +  result * 1.609347F);
```

Comment:
It would be nice if you'd define `result * 1.609347F` as a variable; as for now it's not clear what does this value mean. Same for the next operation.